Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -7130,9 +7130,113 @@
                "chambagri.fr,gb.net,in.ua,notaires.fr,se.com,british-library.uk "
 dble_ext = dble_ext_str.split(",")
 
+# ipwhois exceptions to execution metrics attributes mapping
+# https://ipwhois.readthedocs.io/en/latest/ipwhois.html
+ipwhois_exception_mapping: Dict[Type, str] = {
+
+    # General Errors
+    ipwhois.exceptions.WhoisLookupError: "general_error",
+    ipwhois.exceptions.ASNLookupError: "general_error",
+    ipwhois.exceptions.ASNOriginLookupError: "general_error",
+    ipwhois.exceptions.ASNRegistryError: "general_error",
+    ipwhois.exceptions.ASNParseError: "general_error",
+    ipwhois.exceptions.ASNRegistryError: "general_error",
+    ipwhois.exceptions.BaseIpwhoisException: "general_error",
+    urllib.error.HTTPError: "general_error",
+    ValueError: "general_error",
+    ipwhois.exceptions.IPDefinedError: "general_error",
+
+    # Service Errors
+    ipwhois.exceptions.BlacklistError: "service_error",
+    ipwhois.exceptions.HTTPLookupError: "connection_error",
+
+    # Connection Errors
+    ipwhois.exceptions.NetError: "connection_error",
+
+    # Rate Limit Errors
+    ipwhois.exceptions.HTTPRateLimitError: "quota_error",
+    ipwhois.exceptions.WhoisRateLimitError: "quota_error",
+}
+
+
+def has_rate_limited_result(cmd_results: List[CommandResults]) -> bool:
```

Comment:
Not sure if this function is needed, depends on https://github.com/demisto/content/pull/27370/files#r1245304220